bvm/ballerina-runtime/src/main/resources/MessagesBundle.properties

```diff
@@ -263,3 +263,4 @@ regexp.invalid.unicode.general.category.value = invalid Unicode general category
 regexp.invalid.unicode.property.value = invalid Unicode property value ''{0}''
 regexp.empty.character.class.disallowed = empty character class disallowed
 regexp.invalid.hex.digit = invalid hexadecimal digit
+no.worker.message.received = no worker message received for channel ''{0}''
```

Comment:
Since the concept of channels is not known to the Ballerina user, we can later improve this error message.
 like `no message received from worker ''{0}'' to worker ''{1}''`.

Reply:
The current runtime implementation only depends on the channel name. So we don't store the sender and receiver separately to pass to the error message. 
Needs some modifications at the code generation level. I'll address this in a separate PR.